pageserver/src/tenant/storage_layer/inmemory_layer.rs

```diff
@@ -291,6 +318,10 @@ impl InMemoryLayer {
         Ok(())
     }
 
+    pub async fn put_tombstones(&self, _key_ranges: &[(Range<Key>, Lsn)]) -> Result<()> {
```

Comment:
Although there's lots of existing code that uses unqualified `pub`, as we make changes we're using `pub(crate)` for code that doesn't need to be accessed outside the module.  The neat thing about this is that it will enable the compiler to warn about unused code.

In https://github.com/ctring/neon/pull/16 I've added some pub(crate), and removed a couple of old functions that aren't used any more